### ISSUE TITLE
[0141-linked-list-cycle] Base cases

### ISSUE THREAD

**Issue by yoyotvyoo** [yylou/leetcode-problem-solving#1]
Thank you for the clever solution for [Linked List Cycle](https://github.com/yylou/leetcode/tree/main/0141-linked-list-cycle).

However, I found the lines below are not necessary:

```
# (base case) 
if not head or not head.next: return False 
if head.next == head: return True
```

The while loop will get those base cases covered. ^U^

**Comment by yylou** [comment]
> Thank you for the clever solution for [Linked List Cycle](https://github.com/yylou/leetcode/tree/main/0141-linked-list-cycle).
> 
> However, I found the lines below are not necessary:
> 
> ```
> # (base case) 
> if not head or not head.next: return False 
> if head.next == head: return True
> ```
> 
> The while loop will get those base cases covered. ^U^

You're right man, but you may steel a lil bit runtime on LeetCode online judge system if you highlight this base case at the very first beginning ^o^.
